Specifications/monitoring.md

```diff
@@ -0,0 +1,316 @@
+| `model` | The device model | String | `iPhone15.7`, `Samsung Galaxy S24` |
+| `type` | The device type | `Car`, `Desktop`, `Headset`, `Phone`, `Tablet`, `TV` | `Phone` |
+
+### Media
+
+The `media` JSON data dictionary supports the following keys:
+
+| Field | Description | Format | Examples |
+| - | - | - | - |
+| `asset_url` | The URL of the content being played | String | `https://rts1-lsvs.akamaized.net/out/v1/62441d2399f14dce9e558b5503edba11/index.m3u8` |
+| `id` | A unique media identifier | String | `urn:rts:video:123456` |
+| `metadata_url` | The URL where media metadata was fetched | String | `https://il.srgssr.ch/integrationlayer/2.0/mediaComposition/byUrn/urn:rts:video:3608506` |
+| `origin` | A description of the context in which the media is played | String | `ch.srgssr.app`, `https://www.rts.ch/info/article/123` |
+
+Some remarks:
+
+- Any token appended **by the client** to the URL of the asset being played should not appear in `asset_url`.
+- The `origin` is flexible but should describe the context of playback, for example an application identifier or the URL of the web page hosting the media.
+
+### Operating System
+
+The `os` JSON data dictionary supports the following keys:
+
+| Field | Description | Format | Examples |
+| - | - | - | - |
+| `name` | The operating system name | String | `macOS`, `Windows`, `Android` |
+| `version` | The operating system version | String | `14.5` |
+
+### Player
+
+The `player` JSON data dictionary supports the following keys:
+
+| Field | Description | Format | Examples |
+| - | - | - | - |
+| `name` | The player name | String | `Pillarbox`, `Letterbox`, `video.js` |
+| `platform` | The player platform | `Android`, `Apple`, `Web` | `Android` |
+| `version` | The player version | String | `1.2.3` |
+
+### Quality of Experience Timings
+
+The `qoe_timings` JSON data dictionary supports the following keys:
+
+| Field | Description | Format | Examples |
+| - | - | - | - |
+| `asset` | Time the user waited for asset playback to start | Time in milliseconds | `1145` |
+| `metadata` | Time the user waited for metadata to be retrieved | Time in milliseconds | `412` |
+| `total` | Total time the user waited for playback to start | Time in milliseconds | `1763` |
+
+> [!IMPORTANT]
+> QoE timings measure the perceived user experience. If content preloading in a playlist makes it possible to start instantaneously (or almost), these values should be zero or close to zero.
+
+### Quality of Service Timings
+
+The `qos_timings` JSON data dictionary supports the following keys:
+
+| Field | Description | Format | Examples |
+| - | - | - | - |
+| `asset` | Time for the player to start playing the asset | Time in milliseconds | `1145` |
+| `drm` | Time to load DRM content keys | Time in milliseconds | `245` |
+| `metadata` | Time for metadata to be retrieved by the player | Time in milliseconds | `412` |
+| `token` | Time to fetch an authorization token | Time in milliseconds | `356` |
```

Comment:
We don't send a `total` for `qos_timings`?

Reply:
Nope, it probably does not make sense as processes can be disjoint / superimposed in time. Not sure how a meaningful total could be calculated.